src/contextmenu.js

```diff
@@ -116,6 +124,6 @@ function openInVscode({ linkUrl, selectionText, pageUrl }) {
 
 chrome.contextMenus.create({
     title: 'Open in VSCode',
-    contexts: ['link'],
+    contexts: ['link', 'page'],
```

Comment:
```suggestion
    contexts: ['link'],
```

Again, I'd rather tackle this in a separate PR